src/FSharp.AWS.DynamoDB/Expression/ProjectionExpr.fs

```diff
@@ -62,98 +61,119 @@ type AttributeId with
                     |> InvalidCastException
                     |> raise
 
-            | FParam _ :: _ -> sprintf "internal error; unexpected attribute path '%s'." id.Name |> invalidOp
+            | FParam _ :: _ ->
+                sprintf "internal error; unexpected attribute path '%s'." id.Name
+                |> invalidOp
 
         let ok, prop = ro.TryGetValue id.RootName
-        if ok then 
+
+        if ok then
             let cell = ref null
-            if aux cell id.NestedAttributes prop then value <- cell.Value; true
-            else false
-        else notFound id.NestedAttributes
 
-type ProjectionExpr = 
-    {
-        Attributes : AttributeId []
-        Ctor : Dictionary<string, AttributeValue> -> obj
-    }
+            if aux cell id.NestedAttributes prop then
+                value <- cell.Value
+                true
+            else
+                false
+        else
+            notFound id.NestedAttributes
+
+type ProjectionExpr =
+    { Attributes: AttributeId[]
```

Comment:
can the indents be configured to by in steps of 4? I see this pretty commonly and it matches how `| ` prefixing works on DU cases which is typically nearby, but overall I prefer indenting in blovks of 4 if a formatter is doing it

Reply:
I think that might be what `fsharp_multiline_bracket_style=aligned` does, will try it

Reply:
Hmm, no it doesn't. I think the `cramped` style always effectively has 6 space indents on lines inside multiline brackets. The alternative is go stroustrup or aligned which consumes more vertical space

Reply:
It's all good - thanks for trying....